devel/devel-server.py

```diff
@@ -43,7 +43,11 @@ class MothRequestHandler(http.server.SimpleHTTPRequestHandler):
     
     def __init__(self, request, client_address, server):
         self.directory = str(server.args["theme_dir"])
-        super().__init__(request, client_address, server)
+        try:
+            super().__init__(request, client_address, server, directory=server.args["theme_dir"])
+        except TypeError:
+            super().__init__(request, client_address, server)
+
```

Comment:
I'm starting to wonder if we shouldn't just backport the entire http.server module from Python 3.7.

Reply:
only if you want to march along with upstream backporting all changes to it. :) Can you point to the core problem? Maybe we can put guards around it or "monkeypatch" the behavior out of it (a long-term Pythonic solution).